spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -134,43 +136,48 @@ trait ConformanceExecution extends CommonJobExecution {
                                             fsUtils: FileSystemVersionUtils): Unit = {
     val cmdLineArgs: String = args.mkString(" ")
 
-    PerformanceMetricTools.addJobInfoToAtumMetadata(
-      "conform",
-      preparationResult.pathCfg.standardizationPath,
-      preparationResult.pathCfg.publishPath,
-      menasCredentials.username, cmdLineArgs
-    )
+    // TODO fix for s3 [ref issue #1416]
+//    PerformanceMetricTools.addJobInfoToAtumMetadata(
+//      "conform",
+//      preparationResult.pathCfg.standardizationPath,
+//      preparationResult.pathCfg.publishPath,
+//      menasCredentials.username, cmdLineArgs
+//    )
 
     val withPartCols = result
       .withColumnIfDoesNotExist(InfoDateColumn, to_date(lit(cmd.reportDate), ReportDateFormat))
       .withColumnIfDoesNotExist(InfoDateColumnString, lit(cmd.reportDate))
       .withColumnIfDoesNotExist(InfoVersionColumn, lit(preparationResult.reportVersion))
 
-    val recordCount = result.lastCheckpointRowCount match {
-      case None => withPartCols.count
-      case Some(p) => p
-    }
+    // TODO fix for s3 [ref issue #1416]
+    val recordCount = 100
```

Comment:
-1 would be better if there is no check for it to be > 0. Because 100 seems like a valid value.

Reply:
Thanks, took your advice.